content/en/serverless/libraries_integrations/extension.md

```diff
@@ -19,56 +19,42 @@ The Datadog Lambda Extension is responsible for:
+- `DD_SERVICE`: This sets the `service` tag in Datadog. Use it to group related Lambda functions into a service.
+- `DD_VERSION`: This sets the `version` tag in Datadog. Use it to enable [Deployment Tracking][6].
+- `DD_TAGS`: This sets custom tags in Datadog. Must be a list of `<key>:<value>` separated by commas such as: `layer:api,team:intake`.
 
-2. Add the Datadog Lambda Extension to your container image by adding the following to your Dockerfile:
+If you have the Datadog AWS integration enabled, any AWS resource tag applied to your AWS Lambda function also gets applied in Datadog automatically.
 
-```
-COPY --from=public.ecr.aws/datadog/lambda-extension:<TAG> /opt/extensions/ /opt/extensions
-```
+## Overhead
 
-Replace `<TAG>` with either a specific version number (for example, `{{< latest-lambda-layer-version layer="python" >}}`) or with `latest`. You can see a complete list of possible tags in the [Amazon ECR repository][9].
+Adding Datadog Lambda Extension introduces overhead to your Lambda function's cold starts (that is, the higher init duration), as the Extension needs to initialize. Datadog is continuously working on improvements and treating the work as a high priority.
 
-3. Add the environment variable `DD_API_KEY` and set the value to your Datadog API key on the [API Management page][10]. 
+You may also notice increase of your Lambda function's reported duration, and this is because the Datadog Lambda Extension needs to flush data back to the Datadog API. Although the time spent by the Extension flushing data is reported as part of the duration, it's done *after* AWS returning your function's response back to the client. In other words, the added duration does not slow down your Lambda function. Additional technical details can be found from this [AWS blog post][7].
 
-4. Reference the [sample code][11] to submit a custom metric.
+By default, the Extension sends data back to Datadog at the end of each invocation. This avoids delays of data arrival for sporadic invocations from manual testing and cron jobs. Once the Extension detects a steady and frequent invocation pattern (more than once per minute), it batches data from multiple invocations and flushes periodically at the beginning of the invocation when it's due. This means that *the busier your function is, the lower the average duration overhead per invocation*. 
 
-## Log collection
+For Lambda functions deployed in a region that is far from the Datadog site, for example, a Lambda function deployed in eu-west-1 reporting data to the US1 Datadog site, can observe a higher duration overhead due to the network latency. You can set the environment variable `DD_SERVERLESS_FLUSH_STRATEGY` with value `periodically,60000` on your Lambda functions to flush data once per minute, instead of the default every 10s, and this usually results in a significantly lower *average* duration overhead per invocation.
```

Comment:
are there any other config recommendations we should make for `DD_SERVERLESS_FLUSH_STRATEGY`?

Reply:
Nope, based on my benchmarking results, will recommend flushing every 30s.